ckan/tests/logic/test_validators.py

```diff
@@ -1,3 +1,7 @@
+<<<<<<< HEAD
+
+=======
+>>>>>>> 5021e632b0c0e434e00f100e3c68a1ec4ab9558c
```

Comment:
Hi @amercader can you please suggest as to why this commit hash has been added into the file on its own? 
I did the git push command and while doing so, it showed some error and then again, I had added the file in git add and pushed.

Reply:
@iamarnavgarg looks like a merge error. Just delete these four lines and push again